database/migrations/2023_09_16_023955_add_columns_to_users_table.php

```diff
@@ -14,10 +14,17 @@ class AddColumnsToUsersTable extends Migration
     public function up()
     {
         Schema::table('users', function (Blueprint $table) {
-            $table->string('firstname');
-            $table->string('lastname');
-            $table->string('username')->unique();
+            $table->string('username')->after('name');
+            $table->enum('status', ['active', 'inactive'])->after('password')->default('active');
+            $table->foreignId('role_id')->constrained()->after('status')->nullable();
+            $table->string('address')->after('role_id')->nullable();
+            $table->string('phone_number')->after('address')->nullable();
+            $table->string('image')->after('phone_number')->nullable();
```

Comment:
Replace the string with longText. It should look like this.

$table->longText('image')->after('phone_number')->nullable();

Reply:
I already finshed this.